### PR TITLE
Add default content-type value when setting stream content

### PR DESCRIPTION
**Pull request by Ndiritu**
We missed adding a minor version bump to the previous release by adding a required contentType parameter with no default value causing errors on Graph Core & Graph preview service libs

This adds a default value for the parameter.

@baywet would adding this default match the desired functionality?

### CODE REVIEW

**Review by SilasKenneth — APPROVED** [review]

**Comment by SilasKenneth** [comment]
We can release this alongside #91

**Comment by Ndiritu** [comment]
> We can release this alongside #91

Rebased. Waiting for @baywet's sign-off.

**Review by baywet — APPROVED** [review]

**Pull request merged**
